apps/climatemappedafrica/src/components/HURUmap/PinAndCompare/index.js

```diff
@@ -2,12 +2,14 @@ import { Box, IconButton, SvgIcon } from "@mui/material";
 import PropTypes from "prop-types";
 import React, { useState } from "react";
 
-import useStyles from "./useStyles";
-
 import PinIconDefault from "@/climatemappedafrica/assets/pinBlack.svg";
 import PinIconSelected from "@/climatemappedafrica/assets/pinSelected.svg";
 import Select from "@/climatemappedafrica/components/Select";
 
+// useStyles uses import/definition order to determine how classes are ordered.
+// eslint-disable-next-line import/order
```

Comment:
We'd be better off replacing useStyles with `sx` eslint flat config rules are [slightly different](https://eslint.org/docs/latest/use/configure/rules).

Reply:
... that's what we need to do as part of PesaYetu upgrade i.e. complete migration from `makeStyles`/`useStyles`.